### ISSUE TITLE
loss计算

### ISSUE THREAD

**Issue by zhglzh** [PaddlePaddle/Paddle#6327]
你好，

使用keras-tf计算loss会用到abs\where\less等operator：        
abs_loss = tf.abs(y_true - y_pred)
l1_loss = tf.where(tf.less(abs_loss, 1.0), sq_loss, abs_loss - 0.5)
loss = tf.reduce_sum(l1_loss, -1)

请问paddle如何实现类似操作？ 比如用fluid api，或者直接使用numpy吗？

**Comment by jacquesqiao** [comment]
目前 v2 api中包括 https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/trainer_config_helpers/layers.py#L57 这几种cost，fluid可以实现类似keras的操作，不过有一些op还需要补充和完善。

**Comment by zhglzh** [comment]
有办法用numpy计算吗？

**Comment by zhglzh** [comment]
https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/v2/fluid/tests/book/test_machine_translation.py#L110

http://book.paddlepaddle.org/04.word2vec/index.cn.html

embeddings = parameters.get("_proj").reshape(len(word_dict), embsize)

**Comment by zhglzh** [comment]
如下代码，我在event_handler种修改 event.cost的值，修改后的cost值会在计算梯度时备用到吗？
trainer.train(
    reader=paddle.batch(
        paddle.reader.shuffle(
            create_reader(), buf_size=500),
        batch_size=10),
    feeding=feeding,
    event_handler=**event_handler_plot**,
    num_passes=30)
feeding={'x': 0, 'y': 1}
def **event_handler_plot**(event):
    if isinstance(event, paddle.event.EndIteration):
        _event.cost = 10_
        print(event.cost)
